diff --git a/Directory.Build.props b/Directory.Build.props
index c245ed6..6a73eb1 100644
--- a/Directory.Build.props
+++ b/Directory.Build.props
@@ -6,12 +6,12 @@
     <IsAotCompatible>true</IsAotCompatible>
   </PropertyGroup>
   <PropertyGroup>
-    <AvaloniaVersion>11.0.6</AvaloniaVersion>
-    <AvaloniaBehaviorsVersion>11.0.6</AvaloniaBehaviorsVersion>
-    <AvaloniaEditVersion>11.0.5</AvaloniaEditVersion>
+    <AvaloniaVersion>11.1.3</AvaloniaVersion>
+    <AvaloniaBehaviorsVersion>11.1.0.4</AvaloniaBehaviorsVersion>
+    <AvaloniaEditVersion>11.1.0</AvaloniaEditVersion>
     <AvaloniaEditTextMateGrammarsVersion>0.10.12</AvaloniaEditTextMateGrammarsVersion>
-    <OctokitVersion>9.0.0</OctokitVersion>
-    <CommunityToolkitMvvmVersion>8.2.2</CommunityToolkitMvvmVersion>
+    <OctokitVersion>13.0.1</OctokitVersion>
+    <CommunityToolkitMvvmVersion>8.3.2</CommunityToolkitMvvmVersion>
     <ReactiveMarblesPropertyChangedVersion>2.0.27</ReactiveMarblesPropertyChangedVersion>
     <MicrosoftCodeAnalysisVersion>4.8.0</MicrosoftCodeAnalysisVersion>
   </PropertyGroup>
